MapboxNavigation/RouteVoiceController.swift

```diff
@@ -18,11 +18,6 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
     var fallbackText: String!
     var announcementTimer: Timer?
     
-    /**
-     A boolean value indicating whether instructions should be announced by voice or not.
-     */
-    public var isEnabled: Bool = true
```

Comment:
This is not being used. Setting `NavigationViewController.voiceController` to nil or muting should be sufficient.